### ISSUE TITLE
Modify add node node type may not be properly set

### ISSUE THREAD

**Issue by ibaldin** [RENCI-NRIG/orca5#69]
Reported by Paul. VMs come out default size no matter what you set on the modified node.

**Comment by ibaldin** [comment]
@mcevik0 Mert will check on this and report. Also adding @paul-ruth to the discussion.

**Comment by mcevik0** [comment]
Test:
- Create a slice (RCI - UH)
- Add a node on RCI (Select node type: "Site default"), modify slice

   Result: Node is created with 1 CPU, 512MB memory, 11G disk which corresponds to m1.large as stated in ec2.instance.type property in ec2.site.properties.
   Node information on Flukes (right click) shows "Node Type: Site default"
- Add another node on RCI (Select node type: "XO Extra large"), modify slice

   Result: Node is created with 4 CPUs, 12GB memory, 75G disk which corresponds to xo.xlarge.
   Node information on Flukes (right click) shows "Node Type: Site default" instead of the actual node type.

**Comment by ibaldin** [comment]
So it looks like the node is created properly, but the metadata doesn’t properly reflect its actual size?

-ilya

**Comment by mcevik0** [comment]
I did more tests. It seems just a little bit more complicated. 
- I used the existing slice (RCI - UH) to modify.
- Add a node at another site (TAMU) and connect to UH (Select node type: "XO Extra large"), modify slice.
   Result: Node is created properly, information on Flukes shows node type correctly.
-  Add a node at TAMU site  (Select node type: "XO Extra large"), modify slice.
    Result: Node is created properly, information on Flukes shows node type as "Site default".

It seems there are two different conditions for the metadata to show up on Flukes. 
- When slice is modified for an existing rack, metadata did not reflect actual size.
- When slice is modified to connect a new rack, metadata was correct.
